Jenkinsfile

```diff
@@ -32,8 +32,10 @@ pipeline {
         WALLETCONNECT_PROJECT_ID_PROD = credentials('WALLETCONNECT_PROJECT_ID_PROD')
 	WEB3AUTH_GOOGLE_CLIENT_ID_PROD = credentials('WEB3AUTH_GOOGLE_CLIENT_ID_PROD')
 	WEB3AUTH_GOOGLE_CLIENT_ID_STAGING = credentials('WEB3AUTH_GOOGLE_CLIENT_ID_STAGING')
-	WEB3AUTH_GOOGLE_VERIFIER_PROD = credentials('WEB3AUTH_GOOGLE_VERIFIER_PROD')
-	WEB3AUTH_GOOGLE_VERIFIER_STAGING = credentials('WEB3AUTH_GOOGLE_VERIFIER_STAGING')
+	WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_PROD = credentials('WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_PROD')
+	WEB3AUTH_GOOGLE_VERIFIER_SUB_PROD = credentials('WEB3AUTH_GOOGLE_VERIFIER_SUB_PROD')
+	WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_STAGING = credentials('WEB3AUTH_GOOGLE_VERIFIER_AGGREGATE_STAGING')
+	WEB3AUTH_GOOGLE_VERIFIER_SUB_STAGING = credentials('WEB3AUTH_GOOGLE_VERIFIER_SUB_STAGING')
```

Comment:
We need these four as jenkins credentials

Reply:
please check that those credentials are set in the ci

Reply:
I will add them to Jenkins